tests/utils/test_request_utils.py

```diff
@@ -0,0 +1,32 @@
+import os
+import subprocess
+
+
+def test_request_utils_does_not_import_mlflow():
+    file_content = """
+import importlib.util
+import os
+import sys
+
+file_path = os.path.join(os.path.dirname(__file__), "../../mlflow/utils/request_utils.py")
```

Comment:
```python
import mlflow.utils.request_utils

path = mlflow.utils.request_utils.__file__

file_content = f"
...

file_path = {mlflow.utils.request_utils.__file__!r}

"
```

Reply:
Then you already import mlflow.utils.request_utils, what's the point of the rest? 🤔

Reply:
Ah it's the test! I see!